ideas/172-topic-democracy.md

```diff
@@ -0,0 +1,83 @@
+# Democracy
+
+## Preamble
+
+    Idea: 172-topic-democracy
+    Title: Subjective Delegation Liquid Democracy
+    Status: Draft
+    Created: 2018-04-15
+
+## Summary
+A democracy where you can specify delegate for carbon-voting specific multi level topics or fallback to a predefined defaults if none set.
+
+## Swarm Participants
+- Lead Contributor: Ricardo Guilherme Schmidt
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+This product intend replace authority centralized control for networks with  tokenized participation, such as Status Network currently owned by MultiSig contract 0xBBF0cC1C63F509d48a4674e270D26d80cCAF6022.
+The product would enhance security of network as an attack would require control of more addresses. 
+
+For a democracy where every member understands basic about every subject required for senstive actions inside the network is utopic. 
+As Status Network decisions would be democratic for the users of the platform, a optional voting capability would be given to SNT owners, where they could set their delegate, or vote by themselfs.
+
+When users don't vote and don't delegate, their influence would be delegated to SGT (Status Genesis Tokens) owners consensus.
+
+
+### Product Description
+
+- Users should be able to participate on decisions or delegate their influence. _Important to democracy._
+- Votes are counted as MiniMeToken balance of user at block voting end. _Important to not double tabulating without locking token transfers_ 
```

Comment:
What do you mean by 'not double tabulating' in this context?

Reply:
The user balances should be fixed in a block number, or locked until end of tabulation, otherwise one could move tokens and tabulate many times with same token.